is_core/templatetags/forms.py

```diff
@@ -36,17 +36,33 @@ def fieldset_renderer(context, form, fieldset):
     context_dict.update({
         'class': values.get('class'),
     })
-    if inline_view:
-        return inline_view.render(context, fieldset[0])
-    template = values.get('template') or 'is_core/forms/default_fieldset.html'
-    context_dict.update({
-        'fields': values.get('fields'),
-        'form': form,
-        'title': fieldset[0],
-        'class': values.get('class'),
-        'fieldsets': values.get('fieldsets')
-    })
-    return render_to_string(template, context_dict, request=request)
+
```

Comment:
inline_view, fieldsets and fields can be now in the one deifintion. All will be rendered in the order: fields, fieldsets and inline view